htdocs/class/xoopsform/formselectuser.php

```
@@ -40,17 +40,19 @@ class XoopsFormSelectUser extends XoopsFormElementTray
      *                                 members of that item
      * @param int    $size             Number of rows. "1" makes a drop-down-list.
      * @param bool   $multiple         Allow multiple selections?
+     * @param array  $allowedGroups    Group IDs to restrict users to; empty means no restriction.
+     * @param array  $extraQuery       Extra query parameters appended to the findusers.php popup URL.
      */
-    public function __construct($caption, $name, $includeAnonymous = false, $value = null, $size = 1, $multiple = false)
+    public function __construct($caption, $name, $includeAnonymous = false, $value = null, $size = 1, $multiple = false, array $allowedGroups = [], array $extraQuery = [])
```

PHPMD (2.15.0)

[error] The method __construct has a boolean flag argument $includeAnonymous, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)